app/controllers/join_team_requests_controller.rb

```diff
@@ -3,40 +3,38 @@ class JoinTeamRequestsController < ApplicationController
   # GET /join_team_requests.xml
 
   def action_allowed?
-    current_role_name.eql?("Student")
+    ['Student', 'Instructor', 'Teaching Assistant'].include?(current_role_name) #people with this roles can only access the function provied by the controller 
   end
 
-  def index
-    @join_team_requests = JoinTeamRequest.all
-
-    respond_to do |format|
-      format.html # index.html.erb
-      format.xml  { render :xml => @join_team_requests }
+  private 
```

Comment:
render_request is already declared private in Git. Can you please provide the method names?